docs/getting-started/github-apps/github-apps-troubleshooting-tips.rst

```diff
@@ -54,6 +54,25 @@ you'll need to push another commit after correcting it to activate the sync to D
     - Are all indentation levels correct?
     - Does the name field match, if applicable?
 
+The changes made to my GitHub repo aren't appearing on Dockstore, but I've already installed the GitHub app and made the ``.dockstore.yml`` file. How can I figure out what's going wrong?
+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+If you've already tried waiting a couple of minutes and refreshing the browser on the My Workflows page, you can view GitHub App logs through Dockstore to see if there have been any errors.
+Navigate to the ``/my-workflows`` page and expand the GitHub Organization that the repository belongs to on the left hand side. Then click on the bottom where it says ``See GitHub App Logs``.
+
+.. image:: /assets/images/docs/github-app-logs-button.png
```

Comment:
Comment on  screenshot -- for me the `See GitHub App Logs` and `Refresh Organization` buttons are always on the same line, but maybe because I'm always full-screen. I feel it looks better that way.

Reply:
changed screenshot